oceannavigator/frontend/src/components/TimePicker.jsx

```diff
@@ -18,7 +18,6 @@ import "jquery-ui/../i18n/datepicker-fr.js";
 import "jquery-ui/../i18n/datepicker-fr-CA.js";
 
 const i18n = require("../i18n.js");
-const stringify = require("fast-stable-stringify");
```

Comment:
@htmlboss I don't think this line should have been removed. I believe it is the cause of issue #227